gaas-rest/deploy/helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
The copyright blurb is missing. Can you add it to the file or is it not possible with files of this type?

Reply:
done